test/test.js

```diff
@@ -7,6 +7,10 @@ const path = require('path')
 const BufferList = require('../')
 const { Buffer } = require('buffer')
 
+function tempPath (filename) {
```

Comment:
maybe just use `os.tmpdir()` to make this work nicely

Reply:
done